### DISCUSSION TITLE
How to adjust the log format of error.log

### DISCUSSION THREAD

**Discussion by ouyangli** [apache/apisix#6931] [Q&A]
No configuration found in config file cong/config-default.yaml
![image](https://user-images.githubusercontent.com/8622409/165210880-485f30c3-d3ab-42ba-bdba-fc168e047b26.png)

I want to customize the log format of logs/error.log

Similar to the following settings： 
access_log_format: "$remote_addr [$time_local] $protocol $status $bytes_sent $bytes_received $session_time"

**Comment by tzssangglass** [comment]
I remember that the log format of `error.log` can't be changed, right? cc @spacewander @tokers

**Comment by hansedong** [comment]
The native nginx does not support custom formatting of error_log, so apisix does not support it either.

**Comment by liangmartin** [comment]
我似乎也遇到同样的情况，我想在我设置的路由中加入我的自定义插件，收集它们自己路由产生的error_log保存在指定的路径，就像在nginx设置nginx_conf中conf.d下的error_log，但是失败了，你们可以做到吗
I seem to be encountering the same situation as well. I would like to add my custom plugins to the routes I set up and collect errors generated by their own routes_ Save the log in the specified path, just like setting nginx in nginx_ Error under conf. d in conf_ Log, but it failed. Can you do it
